go.mod

```diff
@@ -5,32 +5,47 @@ go 1.19
 require (
 	github.com/influxdata/influxdb-client-go/v2 v2.12.2
 	github.com/mstoykov/envconfig v1.4.1-0.20220114105314-765c6d8c76f1
-	github.com/sirupsen/logrus v1.9.0
-	github.com/stretchr/testify v1.8.2
-	go.k6.io/k6 v0.45.1
-	gopkg.in/guregu/null.v3 v3.5.0
```

Comment:
Nit; this downgrade caught my attention, but I've seen that `v3.3.0` is the same version that k6 uses, so I guess it's fine/intended.

Reply:
Yes, it is what `go-depsync` suggested.